.github/workflows/chain_interaction.yml

```diff
@@ -65,5 +65,68 @@ jobs:
           ../../desmos/spawn_test_chain.sh -b
           sleep 2 
           ../../desmos/setup_chain.sh
-          cargo test --all-features -- --test-threads=1
+          cargo test --features legacy -- --test-threads=1
+
+  chain_interaction_stargate:
```

Comment:
To reuse the current setup script for tests, we need run another job for stargate version.